Readme.md

```diff
@@ -7,4 +7,8 @@ Current APIs:
 
 - S3
 
-Targeted for Java 1.6+
\ No newline at end of file
+Targeted for Java 1.6+
+
+**Currently, read timeout is disabled to avoid conflicts in updating.**
```

Comment:
Please say "as of version X.Y.Z, read timeout is disabled..." - whatever version this will be released in

Reply:
done